web_src/js/features/repo-issue.js

```diff
@@ -330,16 +345,18 @@ export function initRepoIssueWipTitle() {
 }
 
 export async function updateIssuesMeta(url, action, issueIds, elementId) {
-  return $.ajax({
-    type: 'POST',
-    url,
-    data: {
-      _csrf: csrfToken,
-      action,
-      issue_ids: issueIds,
-      id: elementId,
-    },
-  });
+  try {
+    const params = new URLSearchParams();
+    params.append('action', action);
+    params.append('issue_ids', issueIds);
+    params.append('id', elementId);
+    const response = await POST(url, {data: params});
```

Comment:
```suggestion
    const response = await POST(url, {data: new URLSearchParams({action, issue_ids, id})});
```
And rename function args above to match.

Reply:
Done https://github.com/go-gitea/gitea/pull/29776/commits/f36afb74d942102fcff7ab04289a245c336e5081